### PR TITLE
reference location on docker hub

### PR DESCRIPTION
**Pull request by JordanSussman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,102 @@
...
-      labels:
-        pod-reaper: disabled
-    ...
-    env:
-    - name: EXCLUDED_LABEL_KEY
-      value: pod-reaper
-    - name: EXCLUDED_LABEL_VALUE
-      value: disabled
+# pod-reaper: kills pods dead
+A rules based pod killing container. Pod-Reaper was designed to kill pods that meet specific conditions. For a more information on implemented rules
```
Comment (JordanSussman) [inline README.md]: This sentence seems strange - `For a more information on implemented rules`. I'm assuming that the a is a typo.

Comment (BrianBerzins) [inline README.md]: Yup, good catch.

File: README.md [hunk 2]
```diff
@@ -1,59 +1,102 @@
...
+These environment variables are used to build a label selector to exclude pods from reaping. The key must be a properly formed kubernetes label key. Values are a comma-separated (without whitespace) list of kubernetes label values. Setting exactly one of the key or values environment variables will result in an error.
+
+A pod will be exclude from the pod-reaper if the pod has a metadata label has a key corresponding to the pod-reaper's exclude label key, and that same metadata label has a value in the pod-reaper's list of excluded label values. This means that exclusion requires both the pod-reaper and pod to be configured in a compatible way.
+
+## Implemented Rules
+
+### Chaos Chance
+Flags a pod for reaping based on a random number generator.
+
+Enabled and configured by setting the environment variable `CHAOS_CHANCE` with a floating point value. A random number generator will generate a value in range `[0,1)` and if the the generated value is below the configured chaos chance, the pod will be flagged for reaping.
```
Comment (JordanSussman) [inline README.md]: Talk about the `EXCLUDE_LABEL_KEY` and `EXCLUDE_LABEL_VALUES` again. I.e that you can exclude pods from the chaos chance.

**Comment by BrianBerzins** [comment]
Updated the couple things you mentioned. Let me know what you think!

**Comment by JordanSussman** [comment]
@BrianBerzins everything looks good on a second pass. Are you ready for this to be merged?

**Comment by BrianBerzins** [comment]
Yup! Going to merge

**Pull request merged**